exercises/an_important_rule/problem.md

```diff
@@ -44,22 +44,6 @@ It is worth pointing out that both the synchronous **and** asynchronous
 code have the same problem.  If the rejection handler itself throws
 an error you are going to have a bad time.
 
```

Comment:
This file is also a good place to introduce `.catch`, which you mentioned was lacking.